mscore/tourhandler.cpp

```diff
@@ -427,8 +427,8 @@ void TourHandler::displayTour(Tour* tour)
             mbox->setText(tourMessages[i].message);
 
             // Add "Do not show again" checkbox
-            QCheckBox* showToursBox = new QCheckBox(tr("Do not show me tours"), mbox);
-            showToursBox->setChecked(!showTours);
+            QCheckBox* showToursBox = new QCheckBox(tr("Show tours on startup"), mbox);
```

Comment:
So it turns out this is a lie (just as it is in Edit / Preferences).  This option doesn't just control whether they show on startup, it controls whether they continue to show at all (eg, the tour that appears when you first access the Timeline will appear, or not, based on this setting).  And the startup tour won't appear after you've already seen it, regardless of the state of this option.  So the more accurate working is, "Continuing showing these tours" or something to that effect.

FWIW, I am proposing we add a Help / Tours submenu, with a "Show Tours" checkbox and a "Reset Tours" command that clears the "already seen it" flag from them all.  Not totally convinced we need the "Show tours" option to be exposed in Edit / Preferences any more if we do that, but if we keep it, it should be moved out of "Program Start".

Reply:
Interesting. There's something weird, though, about how you disable tours and click "Next" and it will continue to take you to the next one instead of exiting.

Reply:
Agreed, I noticed the same thing, but then it also occurred to me it would be kind of strange if the "Next" button *didn't* do that.  Probably what should happen is moving the default action (what happens if you press Enter) to "Close", but somehow that feels maybe awkward too.  Anyhow, I say let this stand in your PR as is, I'm working on the further refinements